spec/02-integration/03-db/03-plugins_spec.lua

```diff
@@ -8,50 +8,52 @@ local UUID_PATTERN = "%x%x%x%x%x%x%x%x%-%x%x%x%x%-%x%x%x%x%-%x%x%x%x%-%x%x%x%x%x
 
 
 for _, strategy in helpers.each_strategy() do
-  describe("kong.db [#" .. strategy .. "]", function()
-    local db, bp, service, route
-    local global_plugin
-
-    lazy_setup(function()
-      bp, db = helpers.get_db_utils(strategy, {
-        "routes",
-        "services",
-        "plugins",
-      })
-
-      global_plugin = db.plugins:insert({ name = "key-auth",
-                                          protocols = { "http" },
-                                        })
-      assert.truthy(global_plugin)
-
-    end)
-
-    describe("Plugins #plugins", function()
-
-      before_each(function()
-        service = bp.services:insert()
-        route = bp.routes:insert({ service = { id = service.id },
-                                   protocols = { "tcp" },
-                                   sources = { { ip = "127.0.0.1" } },
-                                 })
+  for default in ipairs{ false, true } do
```

Comment:
I think the loop is using the indices of ipairs instead of the boolean values, so it's currently only testing the default protocol
```suggestion
  for _, default in ipairs{ false, true } do
```